python/lsst/ap/pipe/make_apdb.py

```diff
@@ -108,8 +106,110 @@ def makeApdb(args=None):
     parser = ConfigOnlyParser()
     parsedCmd = parser.parse_args(args=args)
 
-    apdb = parsedCmd.config.diaPipe.apdb.apply(
-        afw_schemas=dict(DiaObject=make_dia_object_schema(),
-                         DiaSource=make_dia_source_schema()))
+    apdb = Apdb(config=parsedCmd.config,
+                afw_schemas=dict(DiaObject=make_dia_object_schema(),
+                                 DiaSource=make_dia_source_schema()))
     apdb.makeSchema()
     return apdb
+
+
+# --------------------------------------------------------------------
+# Copied from pipe_base to let us keep using them after Gen 2 is gone
```

Comment:
Thank you for this comment! Are there no other instances where we need an argument parser like this in gen3? I think putting this here is OK for the purposes of this ticket, but surely other pipetasks must have scripts one runs beforehand... ingestion, perhaps? Whatever it is we do now to declare a directory is a repo? Are other examples such as these just not as inherently configurable?

Reply:
Oops, you're right; the ingestion code uses [a standalone system](https://github.com/lsst/pipe_base/blob/master/python/lsst/pipe/base/configOverrides.py), so half the premise of this ticket is unnecessary. So now I have to rewrite everything. 😭

Reply:
Oh no, ingestion to the... rescue? Something like that. I'll be around whenever you're ready for review take 2. But it will be nice to avoid a surprising bit of argument parsing code living here.

Reply:
Looks like I won't be able to completely avoid having some argument parsing code. The ingestion UI uses [Click](https://github.com/lsst/obs_base/blob/master/python/lsst/obs/base/cli/cmd/commands.py#L84-L101), and I really don't want to figure out how to hook that up to an independent script. 😬 

That said, using the new `ConfigOverrides` class makes the code that needs to live here a _lot_ shorter, so it's still a win.